intermediate_source/torch_compile_tutorial.py

```diff
@@ -36,8 +36,6 @@
 #
 # Note: a modern NVIDIA GPU (Volta or Ampere) is recommended for this tutorial.
 #
-# .. include:: ../beginner_source/new-release-colab.rst
```

Comment:
This file, `torch_compile_tutorial.py`, and the one below, `changing_default_device.py`, don't follow the same pattern as those above. In `torch_compile_tutorial.py`, we're removing `new-release-colab.rst`, but not adding the note. And in `changing_default_device.py`, we add the note, but I don't see us removing `new-release-colab.rst`. Perhaps this is intentional, but thought I should call it out.

Reply:
The note will be appended to the first cell in the Jupyter notebook and will be visible only in the notebook, see changes in conf.py

Reply:
Discussed on a different channel with Svetlana; we are all good hear.